### ISSUE TITLE
WARNING: mergeStatic not found in 'rxjs/operator/merge'

### ISSUE THREAD

**Issue by mikecabana** [isaacplmann/ngx-tour#128]
Application does not break although I get a warning in my console...
```
WARNING in ./node_modules/ngx-tour-md-menu/ngx-tour-md-menu.es5.js
43:23-34 "export 'mergeStatic' was not found in 'rxjs/operator/merge'
```

**Comment by isaacplmann** [comment]
What version of `ngx-tour-md-menu` are you using?  An error very much like this was fixed in #115 on May 17th.

**Comment by mikecabana** [comment]
My bad thought I was on latest. Going from 2.0.0 to 3.1.0 fixed my issue. Closing.
